site/content/docs/5.2/components/modal.md

```diff
@@ -244,6 +244,13 @@ When backdrop is set to static, the modal will not close when clicking outside o
 
 When modals become too long for the user's viewport or device, they scroll independent of the page itself. Try the demo below to see what we mean.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This variant should not be used because it does not respect the Orange Design System specifications.
```

Comment:
Which one? And why?

Reply:
After checking, this callout should be deleted. 
ℹ️ The only thing that is not correct in this PR but that is corrected on `main` by PR #1661 is in the second example (.modal-dialog-scrollable). It's the space between the last sentence of the modal and the divider below.